app/services/search/base.rb

```diff
@@ -37,6 +37,10 @@ def update_mappings(index_alias: self::INDEX_ALIAS)
         Search::Client.indices.put_mapping(index: index_alias, body: self::MAPPINGS)
       end
 
+      def db_count
+        raise "Search classes must implement their own db_count"
+      end
```

Comment:
This isn't used yet. Once a search like `FeedContent` (uses multiple models) is finished, it'll leverage this by adding `db_count` to `Search::FeedContent` with logic to return the appropriate count.